tools/src/main/scala/scala/scalanative/interflow/Inline.scala

```diff
@@ -20,17 +20,17 @@ trait Inline { self: Interflow =>
       .fold[Boolean] {
         false
       } { defn =>
-        val isCtor = originalName(name) match {
+        lazy val isCtor = originalName(name) match {
           case Global.Member(_, sig) if sig.isCtor || sig.isImplCtor =>
             true
           case _ =>
             false
         }
-        val isSmall =
+        lazy val isSmall =
           defn.insts.size <= 8
         val isExtern =
```

Comment:
`isExtern` and other conditions can also be treated with replacement to `lazy val` / `def`. They're also used only once

Reply:
Changed in a1df6de.

I left a lazy val in `shallNot` to make sure that things don't get too slow when logging is enabled.